lms/static/sass/views/_program-details.scss

```diff
@@ -76,3 +76,375 @@
         padding: 0 $full-width-banner-margin/4;
     }
 }
+
+// CSS for April 2017 version of Program Details Page
+
+.program-details {
+    .window-wrap {
+        background-color: $white;
+    }
+}
+.program-details-wrapper {
+
+    .program-details-header {
+        background-color: $light-gray4;
+        @media(min-width: $bp-screen-md) {
+            margin-left: 30px;
+            margin-right: 80px;
+        }
```

Comment:
I would move this down a few lines so it's after the initial margin declarations. Also, an alternative way of writing this is;
```
margin: {
    left: 30px;
    right: 80px;
}
```